sycl/include/sycl/ext/oneapi/matrix/matrix-jit-use.hpp

```diff
@@ -256,6 +257,21 @@ class wi_element {
   wi_element(joint_matrix<T, NumRows, NumCols, Use, Layout, Group> &Mat,
              std::size_t i)
       : M(Mat), idx(i) {}
+
+  std::tuple<size_t, size_t> get_coord() {
```

Comment:
nit: size_t -> uint32_t is probably better
same nit applicable to the code below